src/modules/Library/api/api.ts

```diff
@@ -34,6 +34,6 @@ export const postAppletsToCartApi = (cartItems: PublishedApplet[], signal?: Abor
 
 export const getAppletsFromCartApi = (signal?: AbortSignal) =>
   authApiClient.get('/library/cart', {
-    params: { limit: CART_ITEMS_WITHOUT_LIMIT },
+    params: { limit: MAX_LIMIT },
```

Comment:
please add a TODO to discuss more suitable solution with BA: pagination or infiniti scroll.